src/lib/components/ui/Alert/Alert.jsx

```diff
@@ -59,8 +54,9 @@ Alert.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node,
   ]).isRequired,
+  icon: PropTypes.node,
```

Comment:
There should be `PropTypes.element`. `PropTypes.node` is generic type that can contains everything (like Number, String, Bool, Element, Array)

Reply:
Perhaps we could make it even stricter with `PropTypes.instanceOf(Icon)`? We can always relax this constraint to `PropTypes.element` without introducing breaking change.

Reply:
I'm going for `PropTypes.element` because `Icon` can be a pure function (and default RUI icons _are_ pure functions), `instanceOf` is applicable only for classes.

Reply:
:+1: